css-view-transitions-2/Overview.bs

```diff
@@ -983,6 +1081,75 @@ When capturing the old or new state for an element, perform the following steps
 	Note: This is written in a monkey-patch manner, and will be merged into the algorithm once the L1 spec graduates.
 </div>
 
+## Capturing and applying 'view-transition-group' ## {#vt-group-algorithm}
+
+<div algorithm="additional old capture steps (group)">
+When capturing the old state for an element, perform the following steps given a [=captured element=] |capturedElement|, an [=element=] |element| and a {{ViewTransition}} |transition|:
+
+	1. Let |containingGroupName| be the [=used value=] of |element|'s ''view-transition-group''.
+
+	1. If |containingGroupName| is not null, then:
+
+		1. Let |groupContainerElement| be |transition|'s [=ViewTransition/named elements=][containingGroupName].
+
+		1. If |groupContainerElement| is not null, then:
+
+			1. Let |containerElement| be the old element that is being captured for |containingGroupName|.
+
+			1. If |containerElement| is an ancestor of |element|, then:
+
+				1. Set |capturedElement|'s [=captured element/old transform=] to the transform that would map |element|’s [=border box=] from |containerElement|'s [=border box=] to its current visual position.
+
+				1. Set |capturedElement|'s [=captured element/containing group=] to |containingGroupName|.
+</div>
+
+<div algorithm="additional new capture steps (group)">
+When capturing the new state for an element, perform the following steps given a [=captured element=] |capturedElement| and an [=element=] |element|:
+
+	1. Set |capturedElement|'s [=captured element/containing group=] to the [=used value=] of |element|'s 'view-transition-group'.
+</div>
+
+<div algorithm="additional pseudo-element setup steps (group)">
+When setting up the transition pseudo-element for a [=captured element=] |capturedElement|, given a |transitionName|:
+
+	1. If |capturedElement|'s [=containing group=] is not null, then:
+
+		1. Let |group| be the ''::view-transition-group()'',
+			with its [=view transition =name=] set to |transitionName|.
+
+		1. Let |parentGroup| be the ''::view-transition-group()'',
+			with its [=view transition name=] set to |capturedElement|'s [=containing group=].
+
+		1. Append |group| to |parentGroup|.
+
+	Note: It is TBD how this is resolved when the old and new groups mismatch. See <a href="https://github.com/w3c/csswg-drafts/issues/10631">Issue 10631</a>.
+</div>
+
+<div algorithm="additional pseudo-element style update steps (group)">
+When update the style of the transition pseudo-element for a [=captured element=] |capturedElement|, given a |transitionName| and a {{ViewTransition}} |transition|:
```

Comment:
Same here, a link to the relevant algorithm: https://drafts.csswg.org/css-view-transitions-1/#style-transition-pseudo-elements-algorithm.

Reply:
Done